language/src/modules/Language.rb

```diff
@@ -586,6 +600,21 @@ def ReadSysconfigValues
       nil
     end
 
+    # read UTF-8 status from localed.conf
+    def ReadUtf8Setting
+      return nil if Mode.testsuite
+      # during live installation, we have local configuration
+      if !Stage.initial || Mode.live_installation
+        @localed_conf  = Y2Country.read_locale_conf
+        return nil if @localed_conf.nil?
+        local_lang = @localed_conf["LANG"]
+        @use_utf8 = Builtins.search(local_lang, ".UTF-8") != nil unless local_lang.empty?
```

Comment:
Simplification:
```ruby
local_lang.include?(".UTF-8")
```
But additionally you might need to check for `nil`...


IIRC you can use also the downcase variant and possibly without the dash, e.g. `.utf8`. Should we handle that as well? Command `locale -a` prints only the `.utf8` variant...